### DISCUSSION TITLE
How did you publish your Iconzie documentation in GitHub?

### DISCUSSION THREAD

**Discussion by mak-cs** [FlorianWoelki/obsidian-icon-folder#394] [General]
Hello Florian,

Great Obsidian Plugin! But I have question about your Iconize documentation: What tool did you use, because it look really good. Nice Theme! In all likelihood, you used Obsidian. But how did you publish the documentation on your GitHub page?

It would be great if you could give me a couple trips.

Greetings 
Rainer

**Comment by FlorianWoelki** [comment]
Hey there,

what I use is [Vitepress](https://vitepress.dev/) and to deploy that to GitHub pages, you can take a look at the [CD file](https://github.com/FlorianWoelki/obsidian-iconize/blob/main/.github/workflows/deploy.yml).
